lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1933,6 +1939,8 @@ namespace Js
                         HeapDeleteArray(utf16Len + 1, contents);
 
                         JavascriptOperators::OP_SetProperty(exportObj, propertyRecord->GetPropertyId(), funcObj, this);
```

Comment:
right, at first I told him not to remove it because we didn't know how the format was going to progress.
I think at this point it is safe to simply add it to the `exports` object and remove this line.